### PR TITLE
Switch to using ansible-role-assessment-tool and ansible-role-burp-suite-pro instead of the monolithic ansible-role-kali

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request switches to using [cisagov/ansible-role-assessment-tool](https://github.com/cisagov/ansible-role-assessment-tool) and [cisagov/ansible-role-burp-suite-pro](https://github.com/cisagov/ansible-role-burp-suite-pro) instead of the monolithic [cisagov/ansible-role-kali](https://github.com/cisagov/ansible-role-kali).

As a result of functionality added to [cisagov/ansible-role-assessment-tool](https://github.com/cisagov/ansible-role-assessment-tool) when splitting it out from [cisagov/ansible-role-kali](https://github.com/cisagov/ansible-role-kali), this also adds Python virtual environments for the Python tools installed from source.  See cisagov/ansible-role-assessment-tool#1 for more details.

## 💭 Motivation and Context ##

[cisagov/ansible-role-kali](https://github.com/cisagov/ansible-role-kali) had become too large and monolithic.  It was doing too many things, and it was becoming difficult to add functionality.  It is better for Ansible roles to serve a single purpose, so I never liked having such a monster role.  It made sense to split it up before adding support for Python virtual environments.

## 🧪 Testing ##

I successfully built and deployed a new Kali AMI to our staging COOL environment.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] _All_ future TODOs are captured in issues, which are referenced in code comments.
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.
* [x] Finalize version
* [x] Build a new production Kali AMI using the code in this PR.
* [x] Archive [cisagov/ansible-role-kali](https://github.com/cisagov/ansible-role-kali).

### CODE REVIEW

File: src/assessor_tools.yml [hunk 1]
```diff
@@ -1,28 +1,398 @@
...
+        python2: yes
+        unarchive_extra_opts:
+          - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: "https://github.com/SimplySecurity/SimplyEmail/\
+        tarball/master"
+        install_dir: /tools/SimplyEmail
+        # The misspelling here is intentional.  See
+        # https://github.com/SimplySecurity/SimplyEmail/blob/master/setup/requirments.txt.
```
Comment (dav3r) [inline src/assessor_tools.yml:274]: Helpful comment! 👍

**Review by dav3r — APPROVED** [review]
This looks outstanding!  Thanks for all of the effort you put into this.  Our future selves will be most pleased! 🥇

**Review by dav3r — APPROVED** [review]
Still 👍 👍

File: src/assessor_tools.yml (hunk 1 above)
Comment (mcdonnnj) [reply src/assessor_tools.yml:274]: https://github.com/SimplySecurity/SimplyEmail/pull/49 if that ever got merged (probably very unlikely), we would need to undo the customization.

**Review by mcdonnnj — APPROVED** [review]
Excellent work here. Making assessment tool installation modular is a fantastic addition, and I am always glad to see a move away from a monolithic project. Everything here LGTM ✔

**Review by hillaryj — APPROVED** [review]
🎉

**Pull request merged**